webpack.config.js

```diff
@@ -5,7 +5,8 @@ module.exports = {
   entry: './src/index.js',
   plugins: [
     new HtmlWebpackPlugin({
-      title: 'Output Management',
+      title: 'Todo List',
+      template: './src/index.html',
```

Comment:
- [ ] OPTIONAL 
The `mode` option has not been set in your _webpack_ configuration. Kindly ensure to set the `mode` option for the _webpack_ configuration